src/components/Common/AvatarEditModal.tsx

```diff
@@ -67,18 +67,20 @@ export default function AvatarEditModal({
   const [showCroppedPreview, setShowCroppedPreview] = useState(false);
   const [showCameraPreview, setShowCameraPreview] = useState(false);
 
-  const closeModal = () => {
-    setPreview(undefined);
-    setIsProcessing(false);
-    setIsDeleting(false);
-    setSelectedFile(undefined);
-    setIsCameraOpen(false);
-    setCroppedAreaPixels(null);
-    setCroppedPreview(null);
-    setShowCroppedPreview(false);
-    setShowCameraPreview(false);
-    setCrop({ x: 0, y: 0 });
-    setZoom(1);
+  const closeModal = (force = false) => {
+    if (force) {
+      setPreview(undefined);
+      setIsProcessing(false);
+      setIsDeleting(false);
+      setSelectedFile(undefined);
+      setIsCameraOpen(false);
+      setCroppedAreaPixels(null);
+      setCroppedPreview(null);
+      setShowCroppedPreview(false);
+      setShowCameraPreview(false);
+      setCrop({ x: 0, y: 0 });
+      setZoom(1);
```

Comment:
code doesn't seem readable this way... somethings off...

why not move this to a difference function called `resetState` and call it instead of passing an arg? 🤔 

closeModal should be responsible only on closing the modal right? 🤔